docs/src/main/tut/docs/built-in-supported-types.md

```diff
@@ -5,25 +5,38 @@ title: Built-in Supported Types
 
 ## {{page.title}}
 
-Currently supported types for fields are:
-- `String`, `Boolean`, `Double` (standard
-  and percentage format ending with `%`), `Float` (also supporting percentage),
-    `Int`, `Long`, `Short`, `URL`, `URI`, `Duration`, `FiniteDuration`;
-- [`java.lang.Enum`](https://docs.oracle.com/javase/8/docs/api/java/lang/Enum.html)
-- all collections implementing the `TraversableOnce` trait where the type of
-  the elements is in this list;
-- `Option` for optional values, i.e. values that can or cannot be in the configuration;
-- `Map` with `String` keys and any value type that is in this list;
+PureConfig comes with baked-in support for many types, most of them from the standard Java and Scala libraries. When
+using those types, users don't have to provide anything else in order to be able to use `loadConfig`.
+
+The currently supported basic types are:
+
+- `String`, `Boolean`, `Double` (standard and percentage format ending with `%`), `Float` (also supporting percentage),
+  `Int`, `Long`, `Short`, `URL`, `URI`, `Duration`, `FiniteDuration`;
+- [`java.lang.Enum`](https://docs.oracle.com/javase/8/docs/api/java/lang/Enum.html);
 - everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html) (must be
-  configured first - see [Using Configurable Converters](using-configurable-converters.html));
+  configured first - see [Configurable Converters](configurable-converters.html));
 - [`java.io.File`](https://docs.oracle.com/javase/8/docs/api/java/io/File.html);
 - [`java.util.UUID`](https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html);
 - [`java.nio.file.Path`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html);
-- [`java.util.regex.Pattern`](https://docs.oracle.com/javase/8/docs/api/index.html?java/util/regex/Pattern.html) and [`scala.util.matching.Regex`](https://www.scala-lang.org/files/archive/api/current/scala/util/matching/Regex.html);
-- [`java.math.BigDecimal`](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html), [`java.math.BigInteger`](https://docs.oracle.com/javase/8/docs/api/java/math/BigInteger.html),
-  [`scala.math.BigDecimal`](https://www.scala-lang.org/api/2.12.2/index.html#scala.math.BigDecimal), and [`scala.math.BigInt`](https://www.scala-lang.org/api/2.12.2/index.html#scala.math.BigInt);
+- [`java.util.regex.Pattern`](https://docs.oracle.com/javase/8/docs/api/index.html?java/util/regex/Pattern.html) and
+  [`scala.util.matching.Regex`](https://www.scala-lang.org/files/archive/api/current/scala/util/matching/Regex.html);
+- [`java.math.BigDecimal`](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html),
+  [`java.math.BigInteger`](https://docs.oracle.com/javase/8/docs/api/java/math/BigInteger.html),
+  [`scala.math.BigDecimal`](https://www.scala-lang.org/api/2.12.2/index.html#scala.math.BigDecimal), and
+  [`scala.math.BigInt`](https://www.scala-lang.org/api/2.12.2/index.html#scala.math.BigInt);
```

Comment:
I believe all the links to Scala type around here are either broken or suboptimal.

For example, this one should be https://www.scala-lang.org/api/current/scala/math/BigDecimal.html.